protocol/dubbo/config.go

```diff
@@ -90,6 +90,57 @@ type (
 	}
 )
 
+func GetDefaultClientConfig() ClientConfig {
+	return ClientConfig{
+		ReconnectInterval: 0,
+		ConnectionNum:     16,
+		HeartbeatPeriod:   "5s",
+		SessionTimeout:    "20s",
+		PoolSize:          64,
+		PoolTTL:           600,
+		GrPoolSize:        1200,
+		QueueLen:          64,
+		QueueNumber:       60,
```

Comment:
this size of config may cause higher CPU，this is my config I use better
![image](https://user-images.githubusercontent.com/20589945/65655589-51beae80-e04f-11e9-8a5d-a8ca8378cb59.png)

Reply:
Thank you for sharing your ideas